common/reviews/api/rush-lib.api.md

```diff
@@ -1426,6 +1427,8 @@ export class Subspace {
     // @beta
     contains(project: RushConfigurationProject): boolean;
     // @beta
+    get ensureConsistentVersions(): boolean;
```

Comment:
I've changed it to `Subspace.shouldEnsureConsistentVersions` because if we only kept the CommonVersions class does not have access to the `RushConfiguration`, so we would have to either pass it in to the `CommonVersions.ensureConsistentVersions` or include it in every `CommonVersions` initialization in order to preserve the fallback